docs/field-details.asciidoc

```diff
@@ -3521,288 +3585,2432 @@ example: `/home/alice`
+| TODO
+
+type: keyword
+
+
+
+| custom
+
+// ===============================================================
+
+| process.defense_evasions.memory_sections.protection
+| TODO
+
+type: keyword
+
+
+
+| custom
+
+// ===============================================================
+
+| process.defense_evasions.module_path
+| TODO
+
+type: keyword
+
+
+
+| custom
+
+// ===============================================================
+
+| process.defense_evasions.thread.thread_id
+| TODO
+
+type: long
+
+
+
+| custom
+
+// ===============================================================
+
+| process.defense_evasions.thread.thread_start_address
+| TODO
+
+type: keyword
+
+
+
+| custom
+
+// ===============================================================
+
+| process.defense_evasions.total_memory_size
+| TODO
+
+type: keyword
+
+
+
+| custom
+
+// ===============================================================
+
+| process.domain
+| TODO
+
+type: keyword
+
+
+
+| custom
+
+// ===============================================================
+
+| process.env_variables
+| TODO
+
+type: keyword
+
+
+
+| custom
+
+// ===============================================================
+
+| process.executable
+| Absolute path to the process executable.
+
+type: keyword
+
+Multi-fields:
+
+* process.executable.text (type: text)
+
+
+
+example: `/usr/bin/ssh`
+
+| extended
+
+// ===============================================================
+
+| process.exit_code
+| The exit code of the process, if this is a termination event.
+
+The field should be absent if there is no exit code for the event (e.g. process start).
+
+type: long
+
+example: `137`
+
+| extended
+
+// ===============================================================
+
+| process.gid
+| TODO
+
+type: long
+
+
+
+| custom
+
+// ===============================================================
+
+| process.group
+| TODO
+
+type: keyword
+
+
+
+| custom
+
+// ===============================================================
+
+| process.handle
+| TODO
+
+type: nested
+
+
+
+| custom
+
+// ===============================================================
+
+| process.handle.handle_id
+| TODO
+
+type: long
+
+
+
+| custom
+
+// ===============================================================
+
+| process.handle.handle_name
+| TODO
+
+type: keyword
+
+
+
+| custom
+
+// ===============================================================
+
+| process.handle.handle_type
+| TODO
+
+type: keyword
+
+
+
+| custom
+
+// ===============================================================
+
+| process.has_unbacked_execute_memory
+| TODO
+
+type: boolean
+
+
+
+| custom
+
+// ===============================================================
+
+| process.hash_matched_module
+| TODO
+
+type: boolean
+
+
+
+| custom
+
+// ===============================================================
+
+| process.imphash
+| TODO
+
+type: keyword
+
+
+
+| custom
+
+// ===============================================================
+
+| process.is_sensor
```

Comment:
Consider changing to `is_endpoint`. Sensor is an SMP term not an Elastic Endpoint term.